ZupIT/ritchie-cli
# Adding flags, delete confirmation and new tests to delete repo

### Description

Adding flag support to `rit delete repo` command. Some of the refactoring done as well in this PR:
* Added delete confirmation to `rit delete repo` prompt execution.
* Added tests to support flags and changed tests to work with a tmp directory instead of only mocks.

### How to verify it
* Run `rit delete repo --name=<repo>` and check if has been deleted succesfully.
* Run `rit delete repo` and check if the delete confirmation works.

### Changelog

Added flag support for rit delete repo command
